app/src/main/java/org/wikipedia/search/RecentSearchesFragment.java

```diff
@@ -19,6 +19,7 @@
 import androidx.loader.content.CursorLoader;
 import androidx.loader.content.Loader;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
We prefer to use the `@NonNull` annotation from `androidx.annotation`, instead of the one from `org.jetbrains.annotations`.